src/lang/english.txt

```diff
@@ -3886,7 +3886,7 @@ STR_FINANCES_INFRASTRUCTURE_BUTTON                              :{BLACK}Infrastr
 STR_COMPANY_VIEW_CAPTION                                        :{WHITE}{COMPANY} {BLACK}{COMPANY_NUM}
 STR_COMPANY_VIEW_PRESIDENT_MANAGER_TITLE                        :{WHITE}{PRESIDENT_NAME}{}{GOLD}(Manager)
 
-STR_COMPANY_VIEW_INAUGURATED_TITLE                              :{GOLD}Inaugurated: {WHITE}{NUM}
+STR_COMPANY_VIEW_INAUGURATED_TITLE                              :{GOLD}Inaugurated: {WHITE}{NUM} (period {NUM})
```

Comment:
This should be split into two strings: one containing just `{GOLD}Inaugurated: {WHITE}{NUM}` for calendar timekeeping, and the new string `{GOLD}Inaugurated: {WHITE}{NUM} (period {NUM})` for wallclock timekeeping.

Reply:
Thank you, will do that